test/Microsoft.ApplicationInsights.AspNetCore.Tests/TelemetryInitializers/WebSessionTelemetryInitializerTests.cs

```diff
@@ -1,20 +1,20 @@
-﻿namespace Microsoft.ApplicationInsights.AspNet.Tests.TelemetryInitializers
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Tests.TelemetryInitializers
 {
     using System;
-
-    using Microsoft.ApplicationInsights.AspNet.TelemetryInitializers;
-    using Microsoft.ApplicationInsights.AspNet.Tests.Helpers;
+    using System.Threading.Tasks;
```

Comment:
I am not sure if I understood that right. We need to test initializer in both the frameworks. There was never an ifdefs around this test. Please let me know if I got that wrong.